src/Skyhop.Mail/MailDispatcher.cs

```diff
@@ -14,7 +14,7 @@ namespace Skyhop.Mail
     /// <summary>
     /// Class used to generate and send emails
     /// </summary>
-    public class MailDispatcher
+    public class MailDispatcher : IMailDispatcher
     {
```

Comment:
All xml comments on the implementations of IMailDispatcher can be replaced by &lt;inheritdoc /&gt;